tasks/main.yml

```diff
@@ -77,7 +131,6 @@
   become: True
   when: self_signed_certificate|length > 1
 
-
 - name: upload self-signed cert
```

Comment:
This is unrelated to this PR, but I just noticed this. Who's to say the cert being uploaded here is self-signed? Could be a proper cert as well.